package.json

```diff
@@ -5,6 +5,8 @@
   "main": "app.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
+    "docker-build": "docker build -t content-service .",
+    "docker-run": "sh -c \"docker run -it --rm --name content-service-running -p 80:8080 -e RACKSPACE_USERNAME=$RACKSPACE_USERNAME -e RACKSPACE_APIKEY=$RACKSPACE_APIKEY -e RACKSPACE_REGION=$RACKSPACE_REGION -e RACKSPACE_CONTAINER=$RACKSPACE_CONTAINER -e CONTENT_LOG_LEVEL=$CONTENT_LOG_LEVEL content-service\"",
```

Comment:
Note that doing development orchestration like this this *does* assume that you have a healthy node/npm installation available outside of the container. With that being said, this is mainly for you, so I'd say stick with this if it's convenient. (Also, docker-compose assumes you have a healthy Python/pip installation on your development machine, so it's not that much of an improvement.)